tests/integration/test_dbfs_store/test_dbfs_store.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import uuid
+from pathlib import Path
+
+import pandas as pd
+import pytest
+import yaml
+
+import mlrun
+import mlrun.errors
+from mlrun.utils import logger
+
+here = Path(__file__).absolute().parent
+config_file_path = here / "test-dbfs-store.yml"
+with config_file_path.open() as fp:
+    config = yaml.safe_load(fp)
+
+test_filename = here / "test.txt"
+test_parquet = here / "test_data.parquet"
+test_csv = here / "test_data.csv"
+with open(test_filename, "r") as f:
+    test_string = f.read()
+
+MUST_HAVE_VARIABLES = ["DATABRICKS_TOKEN", "DATABRICKS_WORKSPACE"]
+
+
+def is_dbfs_configured():
+    env_params = config["env"]
+    for necessary_variable in MUST_HAVE_VARIABLES:
+        if env_params.get(necessary_variable, None) is None:
+            return False
+    return True
+
+
+@pytest.mark.skipif(
+    not is_dbfs_configured(),
+    reason="DBFS storage parameters not configured",
+)
+class TestDBFSStore:
+    def setup_method(self):
```

Comment:
I think we might want a cleanup method as well. I know that so far there is just one flow (see my earlier comment about covering other operations as well), but if an assertion fails before the call to `delete()`, we will still need to clean up.

Reply:
Although at least I don't think we'll have a re-entry problem thanks to the use of a UUID.